recipe/meta.yaml

```diff
@@ -150,7 +151,7 @@ outputs:
         - python
         - {{ pin_subpackage('grpc-cpp', exact=True) }}
         - six >=1.6.0
-        - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx and x86_64]
+        - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.14") }}  # [osx and x86_64]
```

Comment:
conda-forge still uses `10.9` here. Is there a reason to force 10.14 as the default? Additionally, the `grpc-cpp` output on conda-forge has `__osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}` in the run section (I can't add a comment there because GH doesn't allow me).

Reply:
Related to that, see https://github.com/conda-forge/grpc-cpp-feedstock/compare/bf30a933d76a05831518bdd680a7f191bca00926..54955c6a2ab7b36f0b40082abbea17cb93f497c5#diff-41024b33205a42b2b33fca0c0d1662ee97f7aff25b9da228ccf417181c2a41dcR51-R53.